be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -242,6 +249,10 @@ void PipelineDriver::finalize(RuntimeState* runtime_state, DriverState state) {
     // Calculate total time before report profile
     _total_timer->update(_total_timer_sw->elapsed_time());
 
+    _schedule_counter->set(driver_acct().get_schedule_times());
+    _schedule_effective_counter->set(driver_acct().get_schedule_effective_times());
+    _schedule_rows_per_chunk->set(driver_acct().get_rows_per_chunk());
+
```

Comment:
Better use Macro `COUNTER_UPDATE` to maintain a consistent style

Reply:
ok.